api/pkg/backend/service.go

```diff
@@ -541,6 +595,24 @@ func (s *APIService) CreateTier(request *restful.Request, response *restful.Resp
 		return
 
 	}
+	//Validation of tenantId of tier:
+	actx := request.Attribute(c.KContext).(*c.Context)
+	token := actx.AuthToken
+	if !ValidateTenant(token, tier.TenantId) {
+		errMsg := fmt.Sprintf("tenantId:%v in request is invalid:", tier.TenantId)
```

Comment:
Change msg to   == > tenantId:%v in request is invalid OR doesn't exist

Reply:
invalid includes doesn't exists